semeio/jobs/correlated_observations_scaling/job_config.py

```diff
@@ -87,6 +87,11 @@ def _min_max_value(value):
     return 0.0 < value < 1.0
 
 
+@configsuite.validator_msg("keys must be provided for CALCULATE_KEYS")
+def _is_not_empty(content):
```

Comment:
This is not a general `_is_not_empty` anymore. Please change the name of the validator :)